transbank/onepay/__init__.py

```diff
@@ -28,10 +28,21 @@ class IntegrationType(Enum):
     TEST = Integration("TEST","https://onepay.ionix.cl","8e279b4e-917d-4cbf-b0e3-9432adefff6a")
     MOCK = Integration("MOCK","https://transbank-onepay-ewallet-mock.herokuapp.com","04533c31-fe7e-43ed-bbc4-1c8ab1538afp")
 
+api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
+shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
+integration_type = IntegrationType.TEST
+callback_url = "http://no.callback.has/been.set"
+app_scheme = None
+qr_width_height = None
+commerce_logo_url = None
+
 class Options(object):
-    def __init__(self, api_key: str, shared_secret: str):
+    def __init__(self, api_key: str, shared_secret: str, qr_width_height=qr_width_height, commerce_logo_url=commerce_logo_url):
```

Comment:
AFAICS, `qr_width_height=None, commerce_logo_url=None` would have the same effect and would be more clear and less verbose.

Reply:
I agree, but, the idea of using those variables instead of directly passing `None` is to have all the default variables configuration in the same place. 
I prefer having them in the same place. What do you think?

Reply:
Fair enough!